doc/user/release-notes.md

```diff
@@ -22,6 +22,8 @@ For information about available versions, see our [Versions page](../versions).
   of floating-point numbers.
 - Support CSV files with headers (`CREATE SOURCE...FORMAT CSV WITH HEADER`).
 - Support naming columns when creating CSV sources.
+- Users can now specify the address and port the daemon binds to on startup.
```

Comment:
Can you throw a link in here to https://materialize.io/docs/overview/configuration/ ?